src/services/web-monitoring-db.js

```diff
@@ -472,3 +478,21 @@ function throwIfError (summary) {
     return data;
   };
 }
+
+function formatCaptureQuery (captureTime) {
```

Comment:
A few thoughts here:

- You’re not handling the previously-used case where `captureTime` is already a formatted string, which is causing the page details view not to load. (Probably easiest to just treat this function as a pass-through for strings.)

- Looks like the date picker returns Moment.js instances, which is the only kind of input you’re handling here. We either:
    - Make sure to also handle `Date` instances or
    - Do an audit to make sure we only ever pass Moment instances in (so we never pass in, say, `someVersion.capture_time`, which is a `Date`).

    It might be easiest to split this up:

    ```js
    function formatDate (date) {
      if (!date) return '';
      else if (moment.isMoment(date)) return date.utc().toISOString();
      else if (date instanceof Date) return date.toISOString();
      else throw new TypeError(`formatDate only takes Moment or Date instances, not ${date.constructor.name}s`);
    }
    function formatDateRange(timeRange) {
      if (typeof timeRange === 'string') return timeRange;

      return `${formatDate(timeRange.startDate)}..${formatDate(timeRange.endDate)}`;
    }
    ```